DataFormats/PatCandidates/interface/MET.h

```diff
@@ -200,11 +247,25 @@ namespace pat {
       // ---- holder for pfMET specific info ---
       std::vector<SpecificPFMETData> pfMET_;
 
+      // MET significance
+      double metSig_;
+      
+      const PackedMETUncertainty findMETTotalShift(MET::METCorrectionLevel cor, MET::METUncertainty shift) const;
+   
+      std::map<MET::METCorrectionLevel, std::vector<MET::METCorrectionType> > corMap_;
+      void initCorMap();
+
+    
+
     protected:
 
       // ---- non-public correction utilities ----
+      //kept for 74X backward-compatibility, not initialized and used in 75X
```

Comment:
It is both. There is a change of format, and in the way the values are filled. 
74X stores directly the full variation (correction + potential uncertainty shift), while in 75X the corrections and uncertainty shifts are considered separately. Plus the addition of  some special cases where the uncertainty shifts does not refer to the general type1 MET central value but e.g. the Type1Smeared central value.